Source/Private/GenerateParameterFile.ps1

```diff
@@ -68,10 +71,20 @@ function GenerateParameterFile {
     
     switch ($PSCmdlet.ParameterSetName) {
         'FromFile' {
-            Out-File -InputObject $ConvertedToJson -FilePath "$($file.DirectoryName)\$filename.parameters.json" -WhatIf:$WhatIfPreference
+            if($AsHashtable.IsPresent) {
+                $parameterBase
+            }
+            else {
+                Out-File -InputObject $ConvertedToJson -FilePath "$($file.DirectoryName)\$filename.parameters.json" -WhatIf:$WhatIfPreference
+            }
         }
         'FromContent' {
-            Out-File -InputObject $ConvertedToJson -FilePath $DestinationPath -WhatIf:$WhatIfPreference
+            if($AsHashtable.IsPresent) {
+                $parameterBase
+            }
+            else {
+                Out-File -InputObject $ConvertedToJson -FilePath $DestinationPath -WhatIf:$WhatIfPreference                
```

Comment:
Since $DestinationPath no longer i mandatory, we need to handle if it is not valid here.